core/examples/luigi-sample-angular/cypress.json

```diff
@@ -8,5 +8,6 @@
   "chromeWebSecurity": false,
   "viewportWidth": 1250,
   "viewportHeight": 790,
-  "baseUrl": "http://localhost:4200/"
+  "baseUrl": "http://localhost:4200/",
+  "numTestsKeptInMemory": 0
```

Comment:
Are you sure we don't need to keep any? By default it's 50, maybe it's better just to reduce the number.